drivers/soundwire/bus.c

```diff
@@ -887,8 +889,8 @@ static int sdw_slave_clk_stop_prepare(struct sdw_slave *slave,
 	ret = sdw_write_no_pm(slave, SDW_SCP_SYSTEMCTRL, val);
 
 	if (ret < 0)
-		dev_err(&slave->dev,
-			"Clock Stop prepare failed for slave: %d", ret);
+		sdw_dev_dbg_or_err(&slave->dev, ret != -ENODATA,
+				   "SDW_SCP_SYSTEMCTRL write ignored:%d\n", ret);
```

Comment:
do we need  \n" in debug message ? The following one doesn't have it

Reply:
thanks @RanderWang yes that's a miss, all debug messages should have a \n

Reply:
@plbossart I don't think we need a `\n`  in the debug message. See [dev_dbg(&pci->dev, "PCI DSP detected");](https://github.com/thesofproject/linux/blob/topic/sof-dev/sound/soc/sof/sof-pci-dev.c#L122), and it works perfectly. 
```
[    3.109702] sof-audio-pci 0000:00:1f.3: PCI DSP detected
```

Reply:
that's not really an explanation @bardliao it's not because others do something that we should do the same error...